### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by SilvaEmerson**
* Refactored API routes
* Created controllers and models module

### CODE REVIEW

**Comment by SilvaEmerson** [comment]
And about the datetime problem that i related to you? Please show me where a must change :sweat_smile:

File: api-mylist/database.py [hunk 1]
```diff
@@ -32,39 +39,30 @@ def read(self, collection, field, value):
...
-    print(a)
-    # db.update('users','ISHxp2DSX1bU1LqED7HW',{'name':'Luís Eduardo'})
-    # db.insert('users', a)
\ No newline at end of file
+db = Database(firebase_admin, firestore)
+
+if __name__ == "__main__":
+    id = db.create_id("users")
+    date = datetime.datetime.now()
+    user = User(id, "edugf", "MASSA", "luiseduardogfranca@gmail.com", datetime.__str__())
```
Comment (luiseduardogfranca) [inline api-mylist/database.py]: Before create a stance for datetime and pass as parameter! 

```
date_creation = datetime.datetime.utcnow()
user = User(id, "edugf", "MASSA", "luiseduardogfranca@gmail.com", date_creation)
```

Comment (SilvaEmerson) [reply api-mylist/database.py]: :white_check_mark:

File: api-mylist/database.py [hunk 2]
```diff
@@ -10,7 +10,7 @@ def __init__(self, firebase_admin, firestore):
         self.firestore = firestore
 
         cred = self.firebase_admin.credentials.Certificate(
-            "./mylist-bfcc8-firebase-adminsdk-clp01-56982f095d.json"
+            "./hello-world-7d62a-firebase-adminsdk-w5pyr-0c74f236dc.json"
```
Comment (luiseduardogfranca) [inline api-mylist/database.py]: Very nice!

**Pull request merged**